Manager/Manager.java

```diff
@@ -1,4 +1,5 @@
 package Manager;
```

Comment:
most the following lines are marked, because IntelliJ changed the indentation; i'll try to mark the actual changes I made.